gtk/src/default/gtk-3.20/_headerbar.scss

```diff
@@ -14,7 +14,7 @@ headerbar,
   }
 
   &:backdrop {
-    border-color: mix(darken($inkstone, 11%), darken($inkstone, 1%), 80%);
+    border-color: $inkstone;
```

Comment:
How about making use of the https://github.com/ubuntu/yaru/blob/684cf87538ac41ea5ac45da309bf0f130daad7b2/gtk/src/default/gtk-3.20/_colors.scss#L85 variable instead? So if you change the headerbar color the border adapts to it
For the backdrop you could add a backdrop variable

Reply:
The goal was to adapt the BG color only and I removed the overly complicated expression for border color with inkstone, which is basically the se, but I agree